models/articles.models.js

```diff
@@ -1,15 +1,24 @@
 const db = require("../db/connection");
 
-exports.selectAllArticles = () => {
+exports.selectAllArticles = (topic) => {
+    const validTopics = ["mitch", "cats", "paper"]
```

Comment:
One tweak that you definitely should consider here - if you're hardcoding the validTopics (not making it dynamic) then what will happen when you use the dev data and the topics are different?

How can we tweak this so that the topics actually come from the database?